tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -640,6 +649,55 @@ EOF
 		chmod g+rx,o+x "${ROOTFS_DIR}"
 	fi
 
+	if [ "${AGENT_POLICY}" == "yes" ]; then
```

Comment:
Seems that there are two ways to inject the policy.
1. Built-in guest image
2. from kata-runtime via annotations

right?